fred2/sexp_tree.cpp

```diff
@@ -3498,6 +3498,12 @@ int sexp_tree::query_default_argument_available(int op, int i)
 			}
 			return 0;
 
+		case OPF_ASTEROID_DEBRIS:
+			if ((Asteroid_info.size() - 3) > 0) {
```

Comment:
Constants or defines are preferred over plain numbers, so this `3` should be `NUM_ASTEROID_POFS`

Reply:
Done